run-vaxrank-b16-test-data.sh

```diff
@@ -4,4 +4,4 @@ vaxrank \
     --vaccine-peptide-length 25 \
     --mhc-predictor netmhc \
     --mhc-alleles H2-Kb,H2-Db \
-    --padding-around-mutation 5
+    --padding-around-mutation 5 --output-ascii-report vaccine-peptides-report.txt
```

Comment:
hilariously enough, I did actually do this and try to push to master - but Gogo gave up on me literally 1 second before the 'git push' went through :) the lack of line break was unbearable when I realized it